lec01-introduction.Rmd

```diff
@@ -2,3 +2,171 @@
+
+```{r}
+ggplot(iris, aes(x=petal_length, fill=species)) +
+    geom_histogram() +
+    facet_wrap(~ species, dir='v')
+```
+
+One last example If you don't have a clear idea of what you are looking for in a data set, maybe you are exploring publicly available data that you have not generated yourself, then it can be useful to visualize the relationships between all the variables to get an overview of the data.
+
+```{r}
+ggpairs(iris, aes(colour = species))
+```
+
+This plot is a bit overwhelming, and the purpose here is not to produce a publication ready figure. You can see that many of the labels are overlapping and not everything fits in the plotting window. We can tweak these properties, which we will get into later in the course. But even in its slightly rough form, this is a powerful data set overview for your own analyses.
+
+Hopefully, these examples have giving you a idea of how simple and powerful it is to use R for you exploratory data analyses. In a few lines of codes we have made some really nice plots, some of which are not even imaginable to create in Excel.
+
+## Saving data
+OK, now you should have a somewhat of an idea of how to generate program... But now that I am done with my analysis, how do I save it? First, what do we want to save? Definitely the plots, those are important. And remember that we said one of the biggest advantages over graphical software is that we have access to all the commands we typed in, and we can save them as an instruction manual for how to make these plots.
+
+Scripts are powerful for automation, but it is still a little bit messy in terms of saving you data, since we will keep the code in one text file and all the output graphs will be saved as separate images. It would not be clear exactly which part of the text file generated what images unless we added notes about this manually. Instead, it would be great if we could keep this all together, in sort of a notebook format... Luckily, there is something called the R Notebook (go to New -> R notebook), which allows us to do exactly that!
+
+# The R Notebook & R Markdown
+In the R Notebook, we are writing in a format called R Markdown, which allows us to combine R code, as the one we were just writing, with the Markdown text format, which allows us to use certain characters to specify headings, bullet points, quotations and even citations. A simple example of how to write in Markdown is to use a single asterisk or underscore to emphasize text _emphasis_ and two asterisks or underscores to strongly emphasize text **strong emphasis**. These will show up as italics and bold typeface when we convert this document to pdf or html, which we will do soon. If you have been writing on online forums, or popular chat services, such as whatsapp, you might already be familiar with this style of writing. In case you haven't seen it before, you have just learnt something about whatsapp in your theoretical ecology class...
```

Comment:
Instead of:
```
A simple example of how to write in Markdown is to use a single asterisk or underscore to emphasize text _emphasis_ and two asterisks or underscores to strongly emphasize text **strong emphasis**.
```

More simply:

```
For example, to **bold** we type out `**bold**` or to *italicize* we type out `*italicize*`
```

RStudio tutorials and docs tend to use single `*` for italicising.

Reply:
Include a link to the cheatsheets or tutorials on RMarkdown.

Reply:
I think it is helpful to think about it as emphasis and strong emphasis though. At least that thinking helps me stick to `*` and `**` instead of involving underlines also.